vizro-core/tests/unit/vizro/models/_controls/test_filter.py

```diff
@@ -219,6 +246,167 @@ def test_filter_isin_date(self, data, value, expected):
         pd.testing.assert_series_equal(result, expected)
 
 
+class TestFilterStaticMethods:
```

Comment:
This is an amazing set of test cases 👏 Really great work.

Reply:
Thanks A!